### ISSUE TITLE
Unable to publish dataset

### ISSUE THREAD

**Issue by aidanmazur** [CSTARS/ckanext-ecosis#26]
Unable to push "Soybean and Aphid Pressure MAZUR TEST 3.0" dataset to search. Getting the following email:
Dear Aidan Mazur,

Your recent push to search for 'Soybean and Aphid Pressure MAZUR TEST 3.0' has failed.  You can try again or contact help@ecospectra.org.  We apologize for the inconvenience

-The EcoSIS Team

Have tried 6 times, tried logging out and in, restarting ecoSIS.

**Comment by aidanmazur** [comment]
Publishing working fine for other datasets. I originally created the problematic dataset as a private one, perhaps that has something to do with the issue?

**Comment by jrmerz** [comment]
Thanks.  Yes, this a backend issue.  Leave the dataset as is and I will take a look.

**Comment by aidanmazur** [comment]
Ok thanks. Spoke too soon about the other dataset though, tried republishing "Soybean Under Aphid Pressure MAZUR TEST 2.0" and got same fail email

**Comment by jrmerz** [comment]
Found the issue.  How did you set the bounding box?

**Comment by aidanmazur** [comment]
The bounding box for the location?

**Comment by jrmerz** [comment]
ya

**Comment by jrmerz** [comment]
All of the points for the box were in the exact same spot.  The geo index for the database didn't like it.  When I drew a box it pushed fine.  I will work on fix, but interested how you got all points at same spot.

**Comment by aidanmazur** [comment]
I uploaded the "Soybean Under Aphid Pressure MAZUR TEST 2.0" dataset from a template for "Productivity and Characterization of Soybean Foliar Traits Under Aphid Pressure" dataset. I then used a templated from ""Soybean Under Aphid Pressure MAZUR TEST 2.0" to create "Soybean and Aphid Pressure MAZUR TEST 3.0" so bounding boxes should have been the same. I might have goofed on something in there. Also, maybe important to note that the original dataset I used the template for (Productivity and Characterization of Soybean Foliar Traits Under Aphid Pressure) and the second (Soybean Under Aphid Pressure MAZUR TEST 2.0) had longitude and latitude in the metadata, while the last one I uploaded (Soybean and Aphid Pressure MAZUR TEST 3.0) didnt, I created different metadata for it, maybe that messed something up
